hack/make/run

```diff
@@ -8,6 +8,8 @@ if ! command -v dockerd &> /dev/null; then
 	false
 fi
 
+DOCKER_COMMAND="$(command -v dockerd)"
+
```

Comment:
Not a blocker, but looking at this one, I see we only use this further down; wondering if we should consider inlining it, e.g.

```bash
dockerd=("$(command -v dockerd)")
```

Or are there use-cases where you'd want to override this through the `DOCKER_COMMAND` variable?

Reply:
The `set -e` behavior of `foo="$(bar)"` vs `foo=("$(bar)")` is that more complex versions of the latter won't necessarily always/consistently exit if `bar` exits non-zero (so it's safer for future changes to keep separated). :see_no_evil:

Reply:
Good to know, thanks!!